packages/react-native-windows-init/src/Cli.ts

```diff
@@ -460,21 +490,24 @@ function isProjectUsingYarn(cwd: string): boolean {
   return !!findUp.sync('yarn.lock', {cwd});
 }
 
-(async () => {
-  if (argv.verbose) {
-    console.log(argv);
+export async function reactNativeWindowsInit(args?: string[]) {
+  args = args ?? process.argv;
```

Comment:
if using process.argv, don't we need to shift 1 or 2 slots to get rid of the `node.exe` and the `scriptname.js` params?

Reply:
No, yargsParser.parse cleans it up.